src/main/java/frc/robot/Robot.java

```diff
@@ -8,6 +8,9 @@
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.TalonFXConfiguration;
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
```

Comment:
Please keep the code clean and remove any unnecessary imports from all files like these. You can tell they are not necessary by the way VSCode highlights the import line.